packages/edit-post/src/components/header/fullscreen-mode-close/style.scss

```diff
@@ -55,5 +55,8 @@
 
 .edit-post-fullscreen-mode-close_site-icon {
 	width: $button-size;
+	height: $button-size;
 	border-radius: $radius-block-ui;
+	background-size: cover;
+	margin-top: -1px;
```

Comment:
Excellent eye. Interestingly, when you're just toggling off that minus 1 margin in the inspector, it makes the site icon look vertically uncentered when it's on:
![margin shift](https://user-images.githubusercontent.com/1204802/147057618-fe7b490c-096b-4c21-b541-b0f172c2bc42.gif)

However when you zoom in and look at the details, you can see that it has to have that negative margin in order to horizontally align with the other buttons in the toolbar. With negative margin:
<img width="328" alt="with minus 1 px" src="https://user-images.githubusercontent.com/1204802/147057666-9c40dbfc-3dd6-43fc-a664-21d25d0a44eb.png">

Without negative margin:

<img width="336" alt="without minus 1 px" src="https://user-images.githubusercontent.com/1204802/147057674-190ef90f-57fa-4946-8a31-dfeb3b5f8871.png">

This _is_, per your instinct, related to the dark square being 61px tall. That math comes $header-height + $border-width, so that the black area covers the gray border below the top-bar.

So which do we keep? I don't have a strong opinion, but I appreciate your attention to detail, so probably do keep it. But I'd add a small comment above, something like `// Compensate for the top-bar border.`, and then change `1px` to `$border-width` instead.

Reply:
Might it be neater to make the black square 60px tall, then add a shadow to hide the grey border? Then we don't need the awkward negative margin.

Reply:
Thanks for the detailed approach here! I had a go at switching to 60px tall and using a shadow to hide the grey border. Unfortunately the styling in the site editor then wound up being a bit unreliable due to the `z-index` stacking order. With `60px` and box-shadow, from time to time the shadow would pop out / a white line would be visible:

![Kapture 2021-12-23 at 11 32 51](https://user-images.githubusercontent.com/14988353/147170919-f4025867-200d-4e28-ab71-8360cd66a694.gif)

I've gone with adding the inline comment and using `$border-width` instead of the hard-coded `1px`, and we can always look at switching to the `60px` approach in a follow-up if we'd like to tweak further.